Домашнее задание Модуль №18

Ulyanov-VN opened the pull request.

Ulyanov-VN committed:
Домашнее задание Модуль №18

sukhoikms27 reviewed (commented):
Плейсхолдеры на новых экранах не соответствуют макетам (вертикальные отступы):

<img width="30%" alt="Image" src="https://github.com/user-attachments/assets/39c11b3a-ae77-4102-a92f-5b7796cc3f14" />

<img width="30%" alt="Image" src="https://github.com/user-attachments/assets/38c769c7-a5f5-440b-999e-a81a0a3228b0" />

макеты:

<img width="324" height="653" alt="Image" src="https://github.com/user-attachments/assets/ffb7d59f-8200-473b-b019-09b50d0f657d" />

<img width="343" height="706" alt="Image" src="https://github.com/user-attachments/assets/ef724c28-ddcd-4586-ad50-6ae8e5e5340d" />

Ulyanov-VN committed:
Домашнее задание Модуль №18 (исправление замечаний)

1. Плейсхолдеры на новых экранах не соответствуют макетам (вертикальные отступы)

Сделал вертикальные отпуступы в соответствии с макетом.
Пришлось создать layout-land, чтобы при перевороте экрана все элепенты помещались на экране.
